### ISSUE TITLE
Crash when loading translation

### ISSUE THREAD

**Issue by AJenbo** [diasurgical/devilutionX#4461]
Only indication of language is the phone model: POCO X3

```
libc.so (memcmp+8)
libdevilutionx.so (offset 0x1000) (LanguagePluralTranslate(char const*, char const*, int))
libdevilutionx.so (offset 0x1000) (devilution::CheckInvHLight())
libdevilutionx.so (offset 0x1000) (devilution::CheckCursMove())
```

**Comment by StephenCWills** [comment]
Could this have been fixed by 122ec0a?

**Comment by AJenbo** [comment]
I don't think so as that was included in 1.4.0

**Comment by StephenCWills** [comment]
It took me entirely too long to figure out how to reproduce this.
